examples/gno.land/p/demo/grc/grc20/types.gno

```diff
@@ -4,17 +4,12 @@ import (
 	"errors"
 	"std"
 
+	"gno.land/p/demo/avl"
 	"gno.land/p/demo/grc/exts"
 )
 
-var (
-	ErrInsufficientBalance   = errors.New("insufficient balance")
-	ErrInsufficientAllowance = errors.New("insufficient allowance")
-	ErrInvalidAddress        = errors.New("invalid address")
-	ErrCannotTransferToSelf  = errors.New("cannot send transfer to self")
-)
-
-type Token interface {
+// GRC20 interface defines the methods that a GRC20 token must implement.
+type GRC20 interface {
```

Comment:
What would you say if this was renamed to `Teller`?

So its usage becomes `grc20.Teller`, rather than the redundant `grc20.GRC20`. This way we clarify that the `Teller` is an "interface" to access a token's ledger, the same way that a bank teller is your "interface" to access your bank account.

Reply:
Sounds good to me. It also eliminates some false expectations that this is the main interface to be used and exposed. In my opinion, it can make sense to expose it, but it makes even more sense to expose Token. This way, people have the choice to use their account from a Teller, and it becomes easier to create other flows.

Reply:
6d763dbe9